src/transformers/convert_slow_tokenizer.py

```diff
@@ -551,6 +551,60 @@ def post_processor(self):
         )
 
 
+class BartphoConverter(SpmConverter):
```

Comment:
I would find it really great if we could avoid having hardcoded values here. Personally, I'll probably see it more clearly once you've answered the two previous questions.

One change you could make to avoid hardcoded values is to replace :
```python
    def unk_id(self, proto):
        unk_id = 3
        return unk_id
```

with 
```python
    def unk_id(self, proto):
        unk_id = self.original_tokenizer.unk_token_id
        return unk_id
```